### PR TITLE
fix(null ids): allowing null ids to be saved.

### PR DESCRIPTION
**Pull request by OrelKaravani**

### CODE REVIEW

**Review by nevobonneh — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]

**Review by ItayBenMoshe — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const ContactQuestioningInfo: React.FC<Props> = (props: Props): JSX.Element => {
                 confirmButtonText: 'כן, המשך'
             }).then((result) => {
                 if (result.value) {
-                    if (checkForDuplicateIds(interactedContact.identificationNumber, interactedContact.id) === -1) {
+                    if (!interactedContact.identificationNumber || checkForDuplicateIds(interactedContact.identificationNumber, interactedContact.id) === -1) {
```
Comment (nevobonneh) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx:49]: Why check duplicate ids returns a number?

Comment (OrelKaravani) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx:49]: It's an existing function. I just fixed the condition to save null ids.

Comment (OrelKaravani) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx:49]: But you're right. It should be fixed to return a boolean value.

Comment (daniel7764) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningInfo.tsx:49]: it returns the result of findIndex

**Pull request merged**